docs/pages/templates/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { MDXRemote } from 'next-mdx-remote';
+import { normalize } from 'path';
+import { Body } from '@ag.ds-next/body';
+import { mapSpacing } from '@ag.ds-next/core';
+import { H2, H3 } from '@ag.ds-next/heading';
+import { Box, Stack } from '@ag.ds-next/box';
+import { Card, CardLink, CardInner } from '@ag.ds-next/card';
+import { mq } from '@ag.ds-next/core';
+import { Text, TextLink } from '@ag.ds-next/text';
+
+import {
+	getMarkdownData,
+	getTemplateList,
+	serializeMarkdown,
+} from '../../lib/mdxUtils';
+import { mdxComponents } from '../../components/utils';
+import { AppLayout } from '../../components/AppLayout';
+import { PageLayout } from '../../components/PageLayout';
+import { DocumentTitle } from '../../components/DocumentTitle';
+
+type StaticProps = Awaited<ReturnType<typeof getStaticProps>>['props'];
+
+export default function TemplatesPage({ source, templateLinks }: StaticProps) {
+	return (
+		<>
+			<DocumentTitle title="Templates | AgDS" />
```

Comment:
```suggestion
			<DocumentTitle title="Templates" />
```


AgDS is appended in the `DocumentTitle`